docs/pages/blog/hello-grid-version-2.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Hello, Grid version 2
```

Comment:
What could we name this piece to try to hook the reader? What could we tell a developer reading our blog or social media feed to convince them that they should click and check out this article?

Reply:
- "Things you should know about the new grid component"
- "Here is how the new grid makes your life easier"
- "Build layout faster with the new grid component"

Any of these catch your attention?

Reply:
I like the last one best:

```suggestion
title: Build layouts faster with the new Grid component
```